config.js

```diff
@@ -1441,6 +1441,23 @@ var config = {
     // dialInConfCodeUrl is the conference mapper converting a meeting id to a PIN used for dial-in
     // or the other way around (more info in resources/cloud-api.swagger)
 
+    // You can use external service for authentication that will redirect back passing a jwt token
+    // You can use tokenAuthUrl config to point to a URL of such service.
+    // The URL for the service supports few params which will be filled in by the code.
+    // tokenAuthUrl:
+    //      'https://myservice.com/auth/{room}?code_challenge_method=S256&code_challenge={code_challenge}&state={state}
+    // {room} - will be replaced with the room name
+    // {code_challenge} - (A web only). A oauth 2.0 code challenge that will be sent to the service. See:
+    // https://datatracker.ietf.org/doc/html/rfc7636. The code verifier will be saved in the sessionStorage
+    // under key: 'code_verifier'.
```

Comment:
Do we want to use some "jitsi_token_auth_" prefix perhaps?

Reply:
I don't see the point, these are params in the link, and it is locally replaced.

Reply:
Ok